README.md

```diff
@@ -114,6 +120,21 @@ There are two options when managing DNS records, manually or ExternalDNS.
 
 Assuming you already have everything needed to install the apps, this is what you need to do.
 
+> [!NOTE]
+> Depending on your infrastructure, you might utilize a Service of type LoadBalancer for the ingress controller. This means you will not have an IP for the domains before installing the ingress controller. After configuring and validating the config, you can install just the ingress controller before the rest of apps with the following command
+>
+> ```bash
+> ./bin/ck8s ops helmfile sc apply -lapp=ingress-nginx --include-transitive-needs
+> ./bin/ck8s ops helmfile wc apply -lapp=ingress-nginx --include-transitive-needs
```

Comment:
I am wondering if this README somewhere and somehow should state that you might not be able to run commands for `wc` if you have not installed Dex in `sc` first, since we usually assume for Welkin that `wc` is configured to point to Dex in `sc` :thinking: